deployment/modules/gcp/cloudbuild/main.tf

```diff
@@ -155,6 +155,12 @@ resource "google_cloudbuild_trigger" "build_trigger" {
       name     = "gcr.io/cloud-builders/gcloud"
       script   = <<EOT
         apt update && apt install jq -y
+
```

Comment:
This will be removed when you're done troubleshooting, right?

Reply:
Yes, I've no idea why the cloud build still attempts to get testdata from the old directory.